### DISCUSSION TITLE
How to create an audio only hls from rtsp stream

### DISCUSSION THREAD

**Discussion by krishgcek** [bluenviron/mediamtx#2382] [Questions] [answered]
Hi All,
Is there way to set audio only for a stream published via rtsp from ffmpeg?

**Comment by aler9** [comment]
<s>Hello, edit the FFmpeg command line in order to publish only the video track and skip the audio one:

```
ffmpeg -re -i ... -map 0:v:0 -c copy -f rtsp rtsp://localhost:8554/mypath
```
</s>

**Reply by krishgcek** [reply]
I need audio only stram. Can I use 0:a:0?
Can I then use it with hls address 8888?

**Reply by aler9** [reply, marked as answer]
Sorry, this is the right command:

```
ffmpeg -re -i ... -map 0:a:0 -c copy -f rtsp rtsp://localhost:8554/mypath
```

Conversion between streaming protocols is automatic, as explained in the README.

[a comment by an automated account is not shown]
